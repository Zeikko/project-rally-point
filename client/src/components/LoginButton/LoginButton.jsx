import React from 'react'
import styled from 'styled-components'

const Heading = styled.h1`
  font-size: 64px;
  text-align: center;
`

function LoginButton() {
  return (
    <a href="/api/auth/login">
      <img src="/img/steam_login.png" alt="Login" />
    </a>
  )
}

export default LoginButton
