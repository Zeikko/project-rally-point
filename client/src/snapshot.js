import renderer from 'react-test-renderer'
import 'jest-styled-components'

export function matchComponentToSnapshot(component) {
  const json = renderer.create(component).toJSON()
  expect(json).toMatchSnapshot()
}