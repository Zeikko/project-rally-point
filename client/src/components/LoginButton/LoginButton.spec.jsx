import React from 'react'
import LoginButton from './LoginButton'
import { matchComponentToSnapshot } from '../../test/snapshot'

describe('LoginButton', () => {
  it('renders correctly', () => {
    matchComponentToSnapshot(<LoginButton />)
  })
})
