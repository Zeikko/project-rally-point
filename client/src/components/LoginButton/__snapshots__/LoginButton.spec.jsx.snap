// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginButton renders correctly 1`] = `
<a
  href="/api/auth/login"
>
  <img
    alt="Login"
    src="/img/steam_login.png"
  />
</a>
`;
